corehq/apps/style/static/dashboard/less/dashboard.less

```diff
@@ -52,6 +52,7 @@
   .panel-heading {
     font-weight: bold;
     font-size: @font-size-large;
+    text-align: center;
```

Comment:
Interesting thought, I agree that randomizing would make a better test than changing things for everyone. I don't know how to tie it to kissmetrics, but can talk with Amelia about it when she's back (unless someone else might know...?)

(I'm assuming you mean this whole PR, not the centering headers in particular)